Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1660,6 +1660,16 @@ - (BOOL)_isWindowResizingEnabled
 
 #endif // HAVE(UIKIT_RESIZABLE_WINDOWS)
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+- (void)_setIsLinkPreview:(BOOL)isLinkPreview
+{
+    WKWEBVIEW_RELEASE_LOG("%p -[WKWebView _setIsLinkPreview] page - %p", self, _page.get());
+    if (_page)
+        _page->enableBuildingInteractionRegions(!isLinkPreview);
```

Comment:
I would expect "setBuildingInteractionRegionsEnabled()" (and I would try to find a way to get "building" out of there -- maybe "setInteractionRegionsEnabled()", or "setShouldBuildInteractionRegions()".

An "enableFoo" taking a bool determining whether it enables or disables is not traditional WebKit style, I think.